easybuild/easyblocks/c/cp2k.py

```diff
@@ -394,32 +397,40 @@ def configure_common(self):
                     raise EasyBuildError("Building the libint wrapper failed")
                 libint_wrapper = '%s/libint_cpp_wrapper.o' % libinttools_path
 
-            # determine LibInt libraries based on major version number
-            libint_maj_ver = get_software_version('LibInt').split('.')[0]
+            # determine Libint libraries based on major version number
+            libint_maj_ver = get_software_version('Libint').split('.')[0]
             if libint_maj_ver == '1':
                 libint_libs = "$(LIBINTLIB)/libderiv.a $(LIBINTLIB)/libint.a $(LIBINTLIB)/libr12.a"
             elif libint_maj_ver == '2':
                 libint_libs = "$(LIBINTLIB)/libint2.a"
             else:
                 raise EasyBuildError("Don't know how to handle libint version %s", libint_maj_ver)
-            self.log.info("Using LibInt version %s" % (libint_maj_ver))
+            self.log.info("Using Libint version %s" % (libint_maj_ver))
 
             options['LIBINTLIB'] = '%s/lib' % libint
             options['LIBS'] += ' %s -lstdc++ %s' % (libint_libs, libint_wrapper)
 
         else:
-            # throw a warning, since CP2K without LibInt doesn't make much sense
-            self.log.warning("LibInt module not loaded, so building without LibInt support")
+            # throw a warning, since CP2K without Libint doesn't make much sense
+            self.log.warning("Libint module not loaded, so building without Libint support")
 
 
         libxc = get_software_root('libxc')
```

Comment:
@damianam please include https://www.cp2k.org/howto:compile#k_libxc_optional_wider_choice_of_xc_functionals as a comment here, to explain what we do here